admin/load.php

```diff
@@ -156,17 +156,38 @@ function perflab_render_modules_page_field( $module_slug, $module_data, $module_
 				<input type="checkbox" id="<?php echo esc_attr( "{$base_id}_enabled" ); ?>" aria-describedby="<?php echo esc_attr( "{$base_id}_description" ); ?>" disabled>
 				<input type="hidden" name="<?php echo esc_attr( "{$base_name}[enabled]" ); ?>" value="<?php echo $enabled ? '1' : '0'; ?>">
 				<?php
+				if ( 'database/sqlite' === $module_slug && ! wp_is_writable( WP_CONTENT_DIR ) ) {
+					printf(
+						/* translators: %s: WP_CONTENT_DIR */
+						__( 'The SQLite module cannot be activated because the %s directory is not writable.', 'performance-lab' ),
+						WP_CONTENT_DIR
+					);
+				} elseif ( 'database/sqlite' === $module_slug && ( ! extension_loaded( 'sqlite3' ) || ! class_exists( 'SQLite3' ) ) ) {
```

Comment:
The `extension_loaded( 'sqlite3' )` check is not present in the `can-load.php` file. I think we need to align them; either we add it in both places or we don't run it in both places.

Reply:
Done 👍